sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -261,8 +263,24 @@ export class StudentService extends RecordDataModelService<Student> {
         id: studentAccountApplicationId,
       } as StudentAccountApplication;
       await entityManager.getRepository(StudentUser).save(studentUser);
+
+      const casSupplier = new CASSupplier();
+      casSupplier.supplierStatus = SupplierStatus.PendingSupplierVerification;
+      casSupplier.supplierStatusUpdatedOn = new Date();
+      casSupplier.isValid = false;
+      casSupplier.creator = auditUser;
+      casSupplier.student = savedStudent;
+      const savedCASSupplier = await entityManager
+        .getRepository(CASSupplier)
+        .save(casSupplier);
+
+      savedStudent.casSupplier = savedCASSupplier;
+      const updatedStudent = await entityManager
+        .getRepository(Student)
+        .save(savedStudent);
```

Comment:
This code should be executed only for new students. Following the same approach from SIN, it can be moved inside the `if` at line 220 (`if (!existingStudent)`), and the update can happen at the same time for SIN and Supplier. Does it make sense?